cmd/sops/main.go

```diff
@@ -200,6 +206,14 @@ func main() {
 			Name:  "set",
 			Usage: `set a specific key or branch in the input JSON or YAML document. value must be a json encoded string. (edit mode only). eg. --set '["somekey"][0] {"somevalue":true}'`,
 		},
+		cli.BoolFlag{
+			Name: "shamir",
```

Comment:
I think `shamir` is clearer. Maybe `shamir-secret-sharing` is even better, although I feel it's a bit too verbose.